pkg/tfgen/docs.go

```diff
@@ -1606,7 +1568,10 @@ func (g *Generator) convertHCLToString(e *Example, hclCode, path, languageName s
 	var diags hcl.Diagnostics
 	var err error
 
-	if cliConverterEnabled() {
+	cache := g.getOrCreateExamplesCache()
+	if r, ok := cache.Lookup(hclCode, languageName); ok {
+		convertedHcl = r
```

Comment:
Are we not returning here because we want 	`g.coverageTracker.languageConversionSuccess(e, languageName, convertedHcl)` to trigger later down?

Reply:
That's right. Looks like this results in a redundant cache.Store call. I can fix that up.